spec/takeaway_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+require './lib/takeaway'
+
+describe Takeaway do
+
+  let(:dish) { double :dish, see: 'Dish output', number: '',
```

Comment:
Style/Blocks: Avoid using {...} for multi-line blocks.<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.